Packer/ArkCase-FOIA/provision.sh

```diff
@@ -32,7 +32,7 @@ cd arkcase-ce/vagrant/provisioning
 move facts.yml .
 git checkout develop
 echo 'localhost ansible_connection=local' > inventory.ini
-ansible-playbook -i inventory.ini -e @facts.yml arkcase-all.yml -t "marketplace, pki, alfresco-ce, pentaho-ee, foia, foia-analytical-reports"
+sudo ansible-playbook -i inventory.ini -e @facts.yml arkcase-all.yml -t "marketplace, pki, alfresco-ce, pentaho-ee, foia, foia-analytical-reports, interactive-reports"
```

Comment:
There should be no need for `sudo` here. Roles that require becoming root have the `become` properties set.

Reply:
I'm agree with you @fabricetriboix , but I was told by @dzdimov that interactive-reports role itself requires sudo when you invoke the playbook